pkg/scalers/azure/azure_data_explorer_test.go

```diff
@@ -17,12 +17,14 @@ limitations under the License.
 package azure
 
 import (
+	"context"
 	"testing"
 
 	"github.com/Azure/azure-kusto-go/kusto/data/errors"
 	"github.com/Azure/azure-kusto-go/kusto/data/table"
 	"github.com/Azure/azure-kusto-go/kusto/data/types"
 	"github.com/Azure/azure-kusto-go/kusto/data/value"
+	kedav1alpha1 "github.com/kedacore/keda/v2/apis/keda/v1alpha1"
```

Comment:
Could you split internal imports in another group? I mean, please, add an empty line before this import just to split imports into "golang", "dependencies" and "internals"